test/inference/gibbs.jl

```diff
@@ -54,6 +54,14 @@
         gibbs = Gibbs(PG(15, :z1, :z2, :z3, :z4), HMC(0.15, 3, :mu1, :mu2))
         chain = sample(MoGtest_default, gibbs, 5_000)
         check_MoGtest_default(chain, atol=0.15)
+
+        for alg in [
+            Gibbs((MH(:s), 10), (HMC(0.2, 4, :m), 10)),
+            Gibbs(MH(:s) => 10, HMC(0.2, 4, :m) => 10),
+        ]
+            chain = sample(gdemo(1.5, 2.0), alg, 5_000)
+            check_numerical(chain, [:s, :m], [49/24, 7/6], atol=0.1)
```

Comment:
Is it necessary to specify the analytical values? I thought there's a function for it already, but maybe I misremember.

Reply:
Found it: https://github.com/TuringLang/Turing.jl/blob/5990fae9f176e84d83bd119fa4a6b0e68f028493/test/test_utils/numerical_tests.jl#L56

```suggestion
            check_gdemo(chain; atol=0.1)
```